src/sentry/seer/code_review/permissions.py

```diff
@@ -0,0 +1,26 @@
+from sentry import features
+from sentry.constants import ENABLE_PR_REVIEW_TEST_GENERATION_DEFAULT, HIDE_AI_FEATURES_DEFAULT
+from sentry.models.organization import Organization
+
+
+def has_code_review_enabled(organization: Organization) -> bool:
```

Comment:
I'm removing the action handling since it is webhook specific:
https://github.com/getsentry/sentry/blob/73825e116e968b38f5bf38eb6dcb59a8965943b3/src/sentry/seer/code_review/webhooks.py#L95-L96

Everything else stays the same.